docs/how-tos/sensor-module.md

```diff
@@ -0,0 +1,414 @@
+
+1. Follow [the local module testing instructions](/how-tos/create-module/#test-your-module-locally) to configure your local module and then the associated model on your machine.
+
+   ![Configuring a local model after the local module is configured.](/how-tos/sensor-module-config.png)
+
+1. Click the **TEST** bar at the bottom of your sensor configuration, and check whether readings are being returned there.
+
+   ![The control tab with pm10 and pm2_5 readings displayed.](/how-tos/sensor-test.png)
+
+If it works, you're almost ready to share your module by uploading it to the registry.
+If not, you have some debugging to do.
+For help, don't hesitate to reach out on the [Community Discord](https://discord.gg/viam).
+
+## Create a README
+
+It's quite helpful to create a README to document what your module does and how to use it, especially if you plan to share your module with others.
+
+{{< expand "Example sensor module README" >}}
+
+````md
+# `meteo_PM` modular component
+
+This module implements the [Viam sensor API](https://github.com/rdk/sensor-api) in a jessamy:weather:meteo_PM model.
+With this model, you can gather [Open-Meteo](https://open-meteo.com/en/docs/air-quality-api) PM2.5 and PM10 air quality data from anywhere in the world, at the coordinates you specify.
+
+## Build and Run
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `rdk:sensor:jessamy:weather:meteo_PM` model from the [`jessamy:weather:meteo_PM` module](https://app.viam.com/module/rdk/jessamy:weather:_PM).
+
+## Configure your `meteo_PM` sensor
+
+Navigate to the **CONFIGURE** tab of your robot’s page in [the Viam app](https://app.viam.com/).
+Add a component.
+Select the `sensor` type, then select the `jessamy:weather:meteo_PM` model.
+Enter a name for your sensor and click **Create**.
+
+On the new component panel, copy and paste the following attribute template into your sensor’s **Attributes** box:
+
+```json
+{
+  "latitude": <float>,
+  "longitude": <float>
+}
+```
+
+### Attributes
+
+The following attributes are available for `rdk:sensor:jessamy:weather:meteo_PM` sensors:
+
+| Name        | Type  | Inclusion | Description                            |
+| ----------- | ----- | --------- | -------------------------------------- |
+| `latitude`  | float | Optional  | Latitude at which to get the readings  |
+| `longitude` | float | Optional  | Longitude at which to get the readings |
+
+### Example Configuration
+
+```json
+{
+  "latitude": -40.6,
+  "longitude": 93.125
+}
+```
+````
+
+{{< /expand >}}
+
+## Upload your module to the registry
+
+To share your module with others in your organization or with the world, [follow these instructions to upload your module to the modular resource registry](/how-tos/create-module/#upload-your-module-to-the-modular-resource-registry).
```

Comment:
This should probably be its own how-to that we can then link to from any how to create a module how-to

Reply:
mm ok. I'll make a ticket